scripts/paternity-test.py

```diff
@@ -0,0 +1,173 @@
+import io
+import os
+import json # might not be needed
+from firecloud import fiss
+from firecloud.errors import FireCloudServerError
+import firecloud.api as fapi
+import numpy as np
+import pandas as pd
+import logging
+#import re <-- doesn't work on Terra
+
+# User-set variables
+BILLING_PROJECT_ID="biodata-catalyst"
```

Comment:
If users are executing this code in a notebook, billing project is already available in the env var `GOOGLE_PROJECT`